publisher_node:
  ros__parameters:
    imu2velo_translation: [-8.086759e-01, 3.195559e-01, -7.997231e-01]
    imu2velo_rotation: [9.999976e-01, 7.553071e-04, -2.035826e-03, -7.854027e-04, 9.998898e-01, -1.482298e-02, 2.024406e-03, 1.482454e-02, 9.998881e-01]
